source/use_case_ndvi_monitor.rst

```diff
@@ -0,0 +1,151 @@
+* The NDVI results are available for streaming to user applications
+* A simple description of how to interpret the NDVI must be available, or the NDVI must be presented
+  in a way that is evident/self-describing
+* A web page must present the latest NDVI and ability to browse back in time (e.g., with a slider)
+
+Post-conditions
+===============
+
+.. The state of the system at the conclusion of the Use Case execution.
+
+* The web page contains information about vegetation state based on the latest Sentinel-2 retrievals
+* The user is able to find fresh vegetation
+
+Normal Flow
+===========
+
+.. Detailed description of the user actions and system responses that will take place during
+   execution of the Use Case under normal, expected conditions. This dialog sequence will ultimately
+   lead to accomplishing the goal stated in the Use Case name and description.
+
+* The user accesses a web page containing information about the current and present vegetation state
+* The user interprets the vegetation state in an extended region (relevant for herding) based on color codes and historic information (using, e.g., a slider)
+
+Alternative Flows
+=================
+
+.. Other, legitimate usage scenarios that can take place within this Use Case.
+
+* The user accesses a mobile app containing information about the current and present vegetation state
+* The user interprets the vegetation state in an extended region (relevant for herding) based on color codes and historic information (using, e.g., a slider)
+
+Exceptions
+==========
+
+.. Anticipated error conditions that could occur during execution of the Use Case, and how the
+   system is to respond to those conditions, or the Use Case execution fails for some reason.
+
+* OPeNDAP access could fail (due to, e.g., high server load?)
+
+Includes
+========
+
+.. Other Use Cases that are included (“called”) by this Use Case (common functionality appearing in
+   multiple Use Cases can be described in a separate Use Case included by the ones that need that
+   common functionality).
+
+* `Investigation of vegetation health variations in a given area based on measurements from Sentinel-2 <ndvi_use_case_investigate>`_
+
+Notes and Issues
+================
+
+.. Additional comments about this Use Case and any remaining open issues that must be resolved. (It
+   is useful to Identify who will resolve each such issue and by what date.)
+
+* We need to make a simple web page to display the results
+* We may want to demonstrate a mobile app
+
+Relevant software
+=================
+
+* Python netCDF4 to access the OPeNDAP stream
+* `Nansat <https://github.com/nansencenter/nansat>`_ could be used as basis for the NDVI tool and to
+  access the Sentinel-2 data stream or any downloaded netCDF-CF files
+
+Relevant datasets
+=================
+
+* NBS Sentinel-2
+
+UML diagram
+===========
+
+.. UML diagram, example;
+
+UML diagram, example;
+
+.. uml::
+
+   @startuml
```

Comment:
Is there a way to view the UML when in revire?

Reply:
review

Reply:
Then I would checkout the branch locally, and compile it as described here: https://s-enda-documentation.readthedocs.io/en/latest/write_docs.html#compiling-the-documentation-locally